pkg/util/apigroup.go

```diff
@@ -71,64 +71,26 @@ func (r *SkippedResourceConfig) Parse(c string) error {
 }
 
 func (r *SkippedResourceConfig) parseSingle(token string) error {
-	switch strings.Count(token, "/") {
-	// Assume user don't want to skip the 'core'(no group name) group.
-	// So, it should be the case "<group>".
-	case 0:
+	parts := strings.Split(token, "/")
+	switch len(parts) {
+	case 1: // Handle case "<group>" and assume user doesn't want to skip the 'core' group
 		r.Groups[token] = struct{}{}
-	// it should be the case "<group>/<version>"
-	case 1:
-		// for core group which don't have the group name, the case should be "v1/<kind>" or "v1/<kind>,<kind>..."
-		if strings.HasPrefix(token, "v1") {
-			var kinds []string
-			for _, k := range strings.Split(token, ",") {
-				if strings.Contains(k, "/") { // "v1/<kind>"
-					s := strings.Split(k, "/")
-					kinds = append(kinds, s[1])
-				} else {
-					kinds = append(kinds, k)
-				}
+		r.GroupVersionKinds[schema.GroupVersionKind{Group: token}] = struct{}{}
```

Comment:
Compared with the original logic, this line is newly added. What is the reason?

Reply:
Sorry, I missed the GroupVersionKinds note.